### DISCUSSION TITLE
Version 1.9: Categories update

### DISCUSSION THREAD

**Discussion by bouncepaw** [bouncepaw/mycorrhiza#142] [General]
This update introduces categories, a new means of organizing hyphae, in addition to links/backlinks and hypha nesting. Starting with this update, you are no longer encouraged to use hypha nesting. In fact, the sibling sidebar is disabled by default now, effectively discouraging it. You can enable it, if your wiki heavily depends on it, though.

=> [Description on the wiki](https://mycorrhiza.wiki/hypha/release/1.9)

<hr /><em>This discussion was created from the release <a href='https://github.com/bouncepaw/mycorrhiza/releases/tag/v1.9.0'>Version 1.9: Categories update</a>.</em>

**Comment by vi** [comment]
Assets are missing in this release.
